automation/run-tests.sh

```diff
@@ -237,7 +237,7 @@ while true; do
         shift
         TEST_TYPE="$1"
         ;;
-    --el7)
+    --el)
```

Comment:
I think that the `8` emphasizes what it is.

Reply:
Changed it to `--el8`